redis-pq-demo/README.md

```diff
@@ -0,0 +1,29 @@
+# Example run-time
+
+This is a demo that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
+
+How to see a full fledged demo, running a local ledger, and a set of agents using the mediator (with redis)  
+
+To run a redis-cluster and acapy as a mediator with redis. 
+```
+git clone https://github.com/hyperledger/aries-mediator-service.git z
+cd redis-pq-demo
+docker-compose -f docker-compose.redis.yml up
+docker-compose up --build
+```
+
+## Example agents
+
+If you want to run a set of agents using the mediator, consider the acapy playground demo. 
+
+```
+git clone https://github.com/hyperledger/aries-cloudagent-python.git
+cd demo/playground
+APP_NETWORK_NAME=mediator_pq docker-compose up
```

Comment:
This network does not exist. As is only one network exists: `redis_cluster`

Reply:
oh, got my ideas mixed up, I'll remove the variable completely as the defaults will work together.